File: Euclid.Storage/Database/Data/Entity/IEntityData.cs
namespace Euclid.Storage.Database.Data
{
    public interface IEntityData
    {
        int Id { get; set; }
        string Name { get; set; }
        string Figure { get; set; }
        string Sex { get; set; }
        string CustomData { get; set; }
    }
}

File: euclid-storage/src/main/java/euclid/storage/database/data/IEntityData.java
package euclid.storage.database.data;

public interface IEntityData {
    int getId();
    void setId(int id);
    String getName();
    void setName(String name);
    String getFigure();
    void setFigure(String figure);
    String getSex();
    void setSex(String sex);
    String getCustomData();
    void setCustomData(String customData);
}
